gallia-prehistoire.csl

```diff
@@ -336,7 +321,7 @@
       <key variable="title"/>
     </sort>
     <layout suffix=".">
-      <group delimiter=" &#8211; ">
+      <group delimiter=" – ">
```

Comment:
please revert - we use escaped characters where it's not immediately visible what character is used.